packages/react-ai/src/components/AIConversation/mocks/mocks.tsx

```diff
@@ -1,40 +1,36 @@
 import React from 'react';
 import { Icon } from '@aws-amplify/ui-react';
-import { Avatars, ConversationMessage, CustomAction } from '../types';
```

Comment:
Are these mocks used in the tests? Should they be in the tests folder?

Reply:
Originaly they were used before we had integration setup, moving to test folder as they are still used there